### ISSUE TITLE
go install error on OpenWrt

### ISSUE THREAD

**Issue by IngwiePhoenix** [charmbracelet/soft-serve#201]
**Describe the bug**
I wanted to install soft-serve on my SBC running OpenWrt (probably the most overkill router ever? 4+4 cores with gpu xD) to synchronize my dotfiles and other important configuration.

However, when I went to install it:
```
root@FriendlyWrt /etc# go install github.com/charmbracelet/soft-serve/cmd/soft@latest
go: downloading github.com/charmbracelet/soft-serve v0.4.4
go: github.com/charmbracelet/soft-serve/cmd/soft@latest (in github.com/charmbracelet/soft-serve@v0.4.4):
        The go.mod file for the module providing named packages contains one or
        more exclude directives. It must not contain directives that would cause
        it to be interpreted differently than if it were the main module.
```

Any idea? I took the command from your readme.

**To Reproduce**
1. Use the "go install" command from the readme.

**Expected behavior**
I expected to see Go build and subsequently install soft-serve.

**Screenshots**
Output above.

**Environment (please complete the following information):**
 - OS: OpenWrt 22.03 (FriendlyWrt fork) (GNU/Linux kernel 5.10.110)
 - Terminal Windows Terminal + ssh
 - Version `@latest`
 - Go version: `1.19`
 - `uname -a`: `Linux FriendlyWrt 5.10.110 #1 SMP Sat Dec 3 01:25:15 CST 2022 aarch64 GNU/Linux`

**Additional context**
I haven't tried to install from git, I just tried to see if this would work out of the box witht he go install command. Makes management a little easier. :)

**Comment by jjg** [comment]
Same error installing from git on Pop!_OS 22.04 LTS with go version go1.17.8 linux/amd64.

```
go install github.com/charmbracelet/soft-serve/cmd/soft@latest
go install: github.com/charmbracelet/soft-serve/cmd/soft@latest (in github.com/charmbracelet/soft-serve@v0.4.4):
        The go.mod file for the module providing named packages contains one or
        more exclude directives. It must not contain directives that would cause
        it to be interpreted differently than if it were the main module.
```

**Comment by mjcc30** [comment]
hello did you try this ?
```
# Debian/Ubuntu
sudo mkdir -p /etc/apt/keyrings
curl -fsSL https://repo.charm.sh/apt/gpg.key | sudo gpg --dearmor -o /etc/apt/keyrings/charm.gpg
echo "deb [signed-by=/etc/apt/keyrings/charm.gpg] https://repo.charm.sh/apt/ * *" | sudo tee /etc/apt/sources.list.d/charm.list
sudo apt update && sudo apt install soft-serve
```

**Comment by aymanbagabas** [comment]
This is now fixed in v0.4.5! You can do `go install github.com/charmbracelet/soft-serve/cmd/soft@latest`
